api/config/config_local.json

```diff
@@ -90,6 +90,11 @@
   "zendesk": {
     "host": "https:\/\/aousupporthelp1634849601.zendesk.com"
   },
+  "absorb": {
+    "enabledForNewUsers": false,
+    "samlIdentityProviderId": "C03q5hexf",
+    "samlServiceProviderId": "133137951028"
```

Comment:
Source: The URL obtained [in these instructions](https://docs.google.com/document/d/1xgcvow0xPL6K4vxyM3PVlW-1E5Ye3X1Y5wZSD5rrPlc/edit?resourcekey=0-q_6-vFevph2piD88EfX-1g#bookmark=id.wesss7shr9s3).